packages/bot-skeleton/src/services/tradeEngine/__tests__/StopStart.js

```diff
@@ -37,6 +36,6 @@ describe('Run Interpreter over bot', () => {
         }, 8000);
     });
     it('Code block is executed correctly', () => {
-        expect(value).to.be.equal(true);
+        expect(value).toBe(true);
```

Comment:
@niloo-fs what do you think about replacing `.to.be.equal` with .`toEqual` everywhere? It seems to me a bit more precise 🙂